olive/evaluator/olive_evaluator.py

```diff
@@ -204,6 +204,7 @@ def generate_metric_user_config_with_model_io(metric: Metric, model: OliveModel)
 
     @staticmethod
     def get_user_config(framework: Framework, data_root: str, metric: Metric):
+        assert metric.user_config, "user_config is not specified in the metric config"
```

Comment:
Where do we insert `user_config` when there is no `user_config` in metric?

Reply:
![image](https://github.com/microsoft/Olive/assets/13343117/0e6fe60c-a85f-49fd-8a4b-9c407850fd77)
This, under `metric.py`